src/utilities/API/Spoonacular.js

```diff
@@ -0,0 +1,8 @@
+const api_key = '&apiKey=5e05f1c45b9b42488369cc8c8cbe13e3';
+export const API_KEY = '5e05f1c45b9b42488369cc8c8cbe13e3';
```

Comment:
these values should be hidden in the .env file. Refer to how Julia did it for Firebase

Reply:
F

Reply:
Fixed, created REACT_APP_SPOONACULAR_API_KEY in .env file and added API key there